docs/howto/create-a-datasource/using-default-datasource.adoc

```diff
@@ -0,0 +1,16 @@
+
+[id='using-default-datasource_{context}']
+= Using the default datasource
+
+[discrete]
+== Prerequisites
+
+* A Maven-based application.
+* A JDBC Driver configured, either auto detected or manually.
+
+[discrete]
+== Procedure
+
+The presence of a JDBC driver will automatically create a datasource with the name `ExampleDS`.
```

Comment:
Only applies for certain set of JDBC drivers. We have a list in the docs (https://docs.thorntail.io/2.2.1.Final/#_autodetectable_drivers), maybe we should put a link here?

Reply:
From looking at the code it looked like installed JDBC drivers, auto or manual, will trigger the default datasource creation

Reply:
Aha! Makes sense, thanks.